intrastat_product/migrations/13.0.1.0.3/post-migration.py

```diff
@@ -2,9 +2,69 @@
 
 from openupgradelib import openupgrade  # pylint: disable=W7936
 
+_months = [
+    (1, "01"),
+    (2, "02"),
+    (3, "03"),
+    (4, "04"),
+    (5, "05"),
+    (6, "06"),
+    (7, "07"),
+    (8, "08"),
+    (9, "09"),
+    (10, "10"),
+    (11, "11"),
+    (12, "12"),
+]
+
+
+def map_intrastat_product_declaration_month(env):
+    openupgrade.map_values(
+        env.cr,
+        openupgrade.get_legacy_name("month"),
+        "month",
+        _months,
+        table="intrastat_product_declaration",
+    )
+
+
+def update_invoice_relation_fields(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_move am
+        SET (intrastat_transaction_id, intrastat_transport_id,
```

Comment:
Is this syntax allowed instead of `field1 = value1, field2 = value2`?

Reply:
see in https://www.postgresql.org/docs/9.5/sql-update.html, there is an example saying "Use the alternative column-list syntax to do the same update:"

Reply:
OK, TIL